lib/tortoise311/connection/backoff.ex

```diff
@@ -4,14 +4,16 @@ defmodule Tortoise311.Connection.Backoff do
   defstruct min_interval: 100, max_interval: 30_000, value: nil
   alias __MODULE__, as: State
 
+  require Logger
+
   @doc """
   Create an opaque data structure that describe a incremental
   back-off.
   """
   def new(opts) do
     min_interval = Keyword.get(opts, :min_interval, 100)
     max_interval = Keyword.get(opts, :max_interval, 30_000)
-
+    Logger.info("[Tortoise311] Backoff intervals are #{min_interval} to #{max_interval}")
```

Comment:
This log message and the next one really feel like debug prints that won't be needed after the backoff fix has been made. It probably would be better to look at the GenServer that holds the backoff state since this is just a data structure and there's no context who's using the backoff here.